samples/HostedInAspNet.Client/HostedInAspNet.Client.csproj

```diff
@@ -1,8 +1,9 @@
-<Project Sdk="Microsoft.NET.Sdk.Razor/2.1.0-preview2-30230">
+<Project Sdk="Microsoft.NET.Sdk.Razor">
```

Comment:
Now that we're using relatively up to date Razor SDK this doesn't generate warnings.